### PR TITLE
fix: bugs 81-82-83-84-85-86-87-88-89-90-91 pbi-30

### PR DESCRIPTION
**Pull request by SebasthianSalpozz**
# Pull Request

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [x] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: projects/client/src/app/components/footer/Footer.tsx [hunk 1]
```diff
@@ -23,43 +45,81 @@ function Footer() {
 
         <div className='mb-4 flex flex-col items-start lg:mb-0 lg:items-start'>
           <p style={{ fontWeight: 'bold' }}>Useful Links</p>
-          <Button className='text-slate-500'>About</Button>
-          <Button className='text-slate-500'>Contact</Button>
-          <Button className='text-slate-500'>Wishlist</Button>
-          <Button className='text-slate-500'>Terms & Conditions</Button>
+          <Button
+            onClick={() => {
```
Comment (rencinasf) [inline projects/client/src/app/components/footer/Footer.tsx:49]: hmmm destructu the onclick function:

` onClick={() => navigate('/about')}
  className='text-slate-500'`

Comment (rencinasf) [reply projects/client/src/app/components/footer/Footer.tsx:49]: maybe you can reuse the class name, it is repeated more than one time

File: projects/client/src/app/components/footer/Footer.tsx [hunk 2]
```diff
@@ -23,43 +45,81 @@ function Footer() {
...
+            Terms & Conditions
+          </Button>
         </div>
 
         <div className='flex flex-col items-start lg:items-start'>
           <p style={{ fontWeight: 'bold' }}>Customer Service</p>
-          <Button className='text-slate-500'>My Account</Button>
-          <Button className='text-slate-500'>My Cart</Button>
+          <Button
+            onClick={() => {
```
Comment (rencinasf) [inline projects/client/src/app/components/footer/Footer.tsx:85]: if it is one line on the onclick function is not needed the {} (same comment as above)

File: projects/client/src/app/pages/termsConditions/TermsConditions.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import InnerLayout from '@/app/layouts/InnerLayout';
+
+function TermsConditions() {
+  return (
+    <InnerLayout>
+      <div className='grid h-full w-full place-content-center '>
+        <h1 className='text-3xl font-bold'>Terms & Conditions Page </h1>
```
Comment (rencinasf) [inline projects/client/src/app/pages/termsConditions/TermsConditions.tsx:7]: hmmm why you are adding an additional space at the end?

File: projects/client/src/app/pages/contact/contact.tsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import InnerLayout from '@/app/layouts/InnerLayout';
+
+function Contact() {
+  return (
+    <InnerLayout>
+      <div className='grid h-full w-full place-content-center '>
+        <h1 className='text-3xl font-bold'>Contact Page </h1>
```
Comment (rencinasf) [inline projects/client/src/app/pages/contact/contact.tsx:7]: I am not totally sure, how this works in react, but I see the same pattern on h1, p, and constant values, so... I think in react you can use const

`const pageTitle = "Contact";
`
or maybe you can use a json:

```
const jsonConstants = {
  pages: { 
    contactPage: "Contact Page"
  }
}
```

Comment (rencinasf) [reply projects/client/src/app/pages/contact/contact.tsx]: hmmm... it is resolved?

**Comment by rencinasf** [comment]
@SebasthianSalpozz, I am not happy with the fixes applied to the code review observations... and if you are happy with this code, go ahead and merge it

**Review by codeFactory12 — APPROVED** [review]

**Pull request merged**